openpype/hosts/maya/api/pipeline.py

```diff
@@ -90,7 +94,9 @@ def install(self):
         register_loader_plugin_path(LOAD_PATH)
         register_creator_plugin_path(CREATE_PATH)
         register_inventory_action_path(INVENTORY_PATH)
-        self.log.info(PUBLISH_PATH)
+
+        # TODO: Expose this via a dedicated `register_template_plugin_path`
```

Comment:
I like this TODO and I think it should be separated PR, merged before this one.

Reply:
How would you like to name the dedicated function for it?

```python
register_workfile_template_builder_placeholder_plugin_path
```
Seems a bit too verbose haha.

Reply:
@iLLiCiTiT any naming preference on this? I can take a look at creating a separate PR for this then.